### PR TITLE
Refactored Checkout and Reorder for Rest-X

### PR DESCRIPTION
**Pull request by SAR2652**
Refactored Checkout and Reorder functions into Flask-RESTX and linted code in all files. Improved routes.py coverage to 98%

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -235,39 +243,81 @@ def post(self):
...
-    """Increases quantity from inventory of a particular item based on the amount specified by user"""
-    app.logger.info(f"Reorder called for product id: {product_id}, condition: {condition}")
-
-    data = request.get_json()
-    existing_record = Inventory.find((product_id, condition))
-    if not existing_record:
-        abort(status.HTTP_404_NOT_FOUND, f"Product with id '{product_id}' was not found.")
-    existing_record.reorder(data)
-    return jsonify(existing_record.serialize()), status.HTTP_200_OK
+# @app.route("/inventory/checkout/<int:product_id>/<string:condition>", methods=["PUT"])
```
Comment (sakshambassi) [inline service/routes.py:299]: do we want to keep or delete this?

File: tests/test_routes.py [hunk 2]
```diff
@@ -215,6 +210,21 @@ def test_list_inventory_records(self):
...
+        expected_records = self._create_inventory_records(2)
+
+        test_record = InventoryFactory()
+        test_record.active = False
+        response = self.client.post(BASE_URL, json=test_record.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        data = response.get_json()
+        self.assertEqual(data['active'], False)
+
+        # test_record = InventoryFactory()
```
Comment (sakshambassi) [inline tests/test_routes.py:222]: if possible, we can clean up comments

**Review by sakshambassi — APPROVED** [review]
great!

**Review by sakshambassi — APPROVED** [review]
awesome

**Review by richa15997 — APPROVED** [review]

**Pull request merged**